app/Main.hs

```diff
@@ -31,8 +25,7 @@ main = do
     IO.hSetEncoding IO.stderr IO.utf8
     IO.hSetBuffering IO.stdout IO.LineBuffering
     IO.hSetBuffering IO.stderr IO.LineBuffering
-    cmd <- Opts.handleParseResult . execParserPure opts =<< getArgs
-    cmd
+    join $ Opts.handleParseResult . execParserPure opts =<< getArgs
```

Comment:
I think this was the only thing in terms of readability where I found it easier to understand the original code rather than the updated code.

Reply:
What's the strength of this opinion—are you objecting or hoping to start a discussion without necessarily objecting?

I personally prefer the updated version—I think `join` immediately communicates that what follows is an `IO` of an `IO`—but I don't mind dropping this commit if this is an actual objection.

Reply:
Mm.... just ignore the comment for now and please merge this in as soon as you can. 

The update is only not as readable to me because I am not as familiar with the code base. If I knew what `handleParseResult` and the other functions do, then the extra 'cmd' isn't necessary. That takes a few seconds to lookup.